### PR TITLE
Moving ra, dec, and W calculations to core

### PR DESCRIPTION
**Pull request by Yash-10**
I noticed the rotational elements were computed in the high-level module. It might be a good choice to move those to core.

Thanks!

### CODE REVIEW

**Review by astrojuanlu — APPROVED** [review]
Thanks a lot @Yash-10 ! While I'm not a super fan of having the functions like this, it's something I have struggled with internally for a long time. It's not possible to do it "the smart way" either because each body uses a different equation essentially. And having a list of parameters that are then used in the equations makes everything just more difficult to read. So I am giving my approval on this, since it's an improvement over what we had before.

**Comment by Yash-10** [comment]
Sorry @astrojuanlu ! I had intended to convert this into a draft since I suspected some problems in the units. I will fix it soon!

Again, sorry for the trouble!

**Comment by astrojuanlu** [comment]
No worries! To clarify, #1229 was not because of this 💪🏽

File: src/poliastro/core/fixed.py [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+    d: float
+        Interval in days from the standard epoch
+
+    Returns
+    -------
+    ra, dec, W: tuple (float)
+        Right ascension and declination of north pole, and angle of the prime meridian.
+
+    """
+    Ja = 99.360714 + 4850.4046 * T
```
Comment (jorgepiloto) [inline src/poliastro/core/fixed.py:192]: This is being computed in degrees and passed in these units to the trigonometric functions below! As pointed out by @Yash-10, the tests within poliastro did not covered all this logic but the validation ones did. Pinging @astrojuanlu, we might discuss this on this week's poliastro meeting (July the 5th, 2021).

Comment (jorgepiloto) [reply src/poliastro/core/fixed.py:192]: The same applies to the rest of the bodies, though this was only noticed for the cases of Jupiter and Moon due to the big amount of terms when computing their rotational elements at epoch.

Comment (Yash-10) [reply src/poliastro/core/fixed.py:192]: Thanks @jorgepiloto ! I will soon open a pull request to (hopefully) rectify my mistake!

Comment (astrojuanlu) [reply src/poliastro/core/fixed.py:192]: Yeah, in general I think having the validation tests is awesome, but also we should have more unit tests :)

**Pull request merged**